[2023-12-16T12:41:36.943+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=manual__2023-12-16T10:56:44.522651+00:00/task_id=Check_Response permission to 509
[2023-12-16T12:41:36.944+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=manual__2023-12-16T10:56:44.522651+00:00/task_id=Check_Response permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=manual__2023-12-16T10:56:44.522651+00:00/task_id=Check_Response'
[2023-12-16T12:41:36.972+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=manual__2023-12-16T10:56:44.522651+00:00/task_id=Check_Response permission to 509
[2023-12-16T12:41:36.973+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=manual__2023-12-16T10:56:44.522651+00:00/task_id=Check_Response permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=manual__2023-12-16T10:56:44.522651+00:00/task_id=Check_Response'
[2023-12-16T12:41:37.007+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Check_Response manual__2023-12-16T10:56:44.522651+00:00 [queued]>
[2023-12-16T12:41:37.018+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Check_Response manual__2023-12-16T10:56:44.522651+00:00 [queued]>
[2023-12-16T12:41:37.019+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 3
[2023-12-16T12:41:37.035+0000] {taskinstance.py:1382} INFO - Executing <Task(BranchPythonOperator): Check_Response> on 2023-12-16 10:56:44.522651+00:00
[2023-12-16T12:41:37.040+0000] {standard_task_runner.py:57} INFO - Started process 1903 to run task
[2023-12-16T12:41:37.043+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Check_Response', 'manual__2023-12-16T10:56:44.522651+00:00', '--job-id', '144', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmpica__yur']
[2023-12-16T12:41:37.045+0000] {standard_task_runner.py:85} INFO - Job 144: Subtask Check_Response
[2023-12-16T12:41:37.098+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=manual__2023-12-16T10:56:44.522651+00:00/task_id=Check_Response permission to 509
[2023-12-16T12:41:37.099+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=manual__2023-12-16T10:56:44.522651+00:00/task_id=Check_Response permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=manual__2023-12-16T10:56:44.522651+00:00/task_id=Check_Response'
[2023-12-16T12:41:37.102+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Check_Response manual__2023-12-16T10:56:44.522651+00:00 [running]> on host d739bc9bef06
[2023-12-16T12:41:37.216+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Check_Response' AIRFLOW_CTX_EXECUTION_DATE='2023-12-16T10:56:44.522651+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-16T10:56:44.522651+00:00'
[2023-12-16T12:41:37.231+0000] {airQuality.py:185} INFO - Data Found: {'city': ['Jakarta'], 'country': ['ID'], 'unit': ['pm25'], 'last_value': [-999.0], 'lat': [-6.182536], 'lng': [106.834236]}
[2023-12-16T12:41:37.232+0000] {python.py:194} INFO - Done. Returned value was: Transform
[2023-12-16T12:41:37.233+0000] {python.py:227} INFO - Branch callable return Transform
[2023-12-16T12:41:37.234+0000] {skipmixin.py:173} INFO - Following branch Transform
[2023-12-16T12:41:37.254+0000] {skipmixin.py:239} INFO - Skipping tasks [('Error_Response', -1)]
[2023-12-16T12:41:37.301+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=air_quality, task_id=Check_Response, execution_date=20231216T105644, start_date=20231216T124137, end_date=20231216T124137
[2023-12-16T12:41:37.337+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-16T12:41:37.374+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
